### PR TITLE
Standardise order of variables

### PR DESCRIPTION
**Pull request by jetlfj**
The order of variables of the Internship class are not the same across related classes and testcases.

Standardising them will allow for easier formatting and less bug-prone code.

Let's follow the format below:
* Name
* Position
* Status
* Phone
* Email
* Address
* Tasks
* Tags
* Remarks

### CODE REVIEW

File: src/test/java/seedu/intrack/testutil/TypicalInternships.java [hunk 1]
```diff
@@ -30,60 +30,52 @@
...
-            .withAddress("little india").build();
-    public static final Internship IDA = new InternshipBuilder().withName("Ida Mueller").withPhone("8482131")
-            .withPosition("Data Engineer").withEmail("hans@example.com").withStatus("Offered")
-            .withAddress("chicago ave").build();
+    public static final Internship HOON = new InternshipBuilder().withName("Hoon Meier")
+            .withPosition("Product Designer").withStatus("Progress").withPhone("8482424")
+            .withEmail("stefan@example.com").withAddress("little india").build();
+    public static final Internship IDA = new InternshipBuilder().withName("Ida Mueller")
+            .withPosition("Data Engineer").withEmail("hans@example.com").withPhone("8482131")
+            .withStatus("Progress").withAddress("chicago ave").build();
```
Comment (chantellyu) [inline src/test/java/seedu/intrack/testutil/TypicalInternships.java:69]:
```suggestion
            .withPosition("Data Engineer").withStatus("Progress").withPhone("8482131")
            .withEmail("hans@example.com").withAddress("chicago ave").build();
```

File: src/main/java/seedu/intrack/model/internship/Internship.java [hunk 2]
```diff
@@ -125,18 +123,18 @@ public boolean equals(Object other) {
         Internship otherInternship = (Internship) other;
         return otherInternship.getName().equals(getName())
                 && otherInternship.getPosition().equals(getPosition())
+                // && otherInternship.getStatus().equals(getStatus())
```
Comment (chantellyu) [inline src/main/java/seedu/intrack/model/internship/Internship.java:126]: Would it be better to also check for equality in status?

File: src/main/java/seedu/intrack/model/internship/Internship.java [hunk 3]
```diff
@@ -125,18 +123,18 @@ public boolean equals(Object other) {
...
                 && otherInternship.getPhone().equals(getPhone())
                 && otherInternship.getEmail().equals(getEmail())
-                && otherInternship.getStatus().equals(getStatus())
                 && otherInternship.getAddress().equals(getAddress())
-                && otherInternship.getTasks().equals(getTasks())
+                // && otherInternship.getTasks().equals(getTasks())
```
Comment (chantellyu) [inline src/main/java/seedu/intrack/model/internship/Internship.java:130]: Would it be better to also check for equality in tasks?

**Review by chantellyu — COMMENTED** [review]
Good job picking up on issues such as missing testcases in the code and properly formatting all the code, just a minor nitpick

File: src/main/java/seedu/intrack/model/internship/Internship.java (hunk 2 above)
Comment (jetlfj) [reply src/main/java/seedu/intrack/model/internship/Internship.java:126]:
```suggestion
                && otherInternship.getStatus().equals(getStatus())
```

File: src/main/java/seedu/intrack/model/internship/Internship.java (hunk 3 above)
Comment (jetlfj) [reply src/main/java/seedu/intrack/model/internship/Internship.java:130]:
```suggestion
                && otherInternship.getTasks().equals(getTasks())
```

File: src/main/java/seedu/intrack/model/internship/Internship.java (hunk 2 above)
Comment (jetlfj) [inline src/main/java/seedu/intrack/model/internship/Internship.java:126]:
```suggestion
                && otherInternship.getStatus().equals(getStatus())
```

**Pull request merged**